src/actions.ts

```diff
@@ -34,6 +34,10 @@ export const RequestActionObjectResolver = makeEnum({
   SaleOptOut: 'SALE_OPT_OUT',
   /** A tracking opt out request */
   TrackingOptOut: 'TRACKING_OPT_OUT',
+  /** Opt-in to the sale of personal data */
+  SaleOptIn: 'SALE_OPT_IN',
+  /** A tracking opt in request */
+  TrackingOptIn: 'TRACKING_OPT_IN',
```

Comment:
Is the idea that there would metadata about what type of tracking are opted in with the TRACKING_OPT_IN action?

Reply:
@eligrey ya thats the idea. the customer can create multiple workflows and have different workflows map to different opt in actions